duckduckgo/Android
# Android: Align codestyle with ktlint checks

Task/Issue URL: https://app.asana.com/0/0/1176540122453977/f
Tech Design URL: 
CC: 

**Description**:
Update code style settings with our ktlint rules
Update documentation to explain steps

**Steps to test this PR**:
1. Autoformat your app/src/main/java folder with new settings
1. run ktlint and ensure it doesn't fail


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
